doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,269 @@
+that is available at the MPI for Biogeochemistry in Jena: `cmorize_obs_mte.py
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmorize_obs_mte.py>`_.
+
+All the necessary information about the dataset to write the filename
+correctly, and which variable is of interest, is stored in a seperate
+configuration file: `MTE.yml
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmor_config/MTE.yml>`_
+in the directory ``ESMValTool/esmvaltool/cmorizers/obs/cmor_config/``. Note
+that the name of this configuration file has to be identical to the name of
+your data set.
+
+The first part of this configuration file defines the filename of the raw
+observations file, the second part defines the common global attributes for 
+the cmorizer output, e.g. information that is needed to piece together the 
+final observations file name in the correct structure (see Section 6). The 
+third part defines the variables that are supposed to be cmorized.
+
+The actual cmorizing script ``cmorize_obs_mte.py`` consists of a header with
+information on where and how to download the data, and noting the last access 
+of the data webpage. 
+
+The main body of the CMORizer script must contain a function called
+
+.. code-block:: python
+
+   def cmorization(in_dir, out_dir, cfg):
+
+with this exact call signature. Here, ``in_dir`` corresponds to the input 
+directory of the raw files, ``out_dir`` to the output directory of final 
+reformatted data set and ``cfg`` to the configuration dictionary given by 
+the  ``.yml`` configuration file. This function needs to return ``None``. All 
+the work, i.e. loading of the raw files, processing them and saving the final 
+output, has to be performed inside its body. To simplify this process, ESMValTool 
+provides a set of predefined utilities.py_, which can be imported into your CMORizer 
+by
+
+.. code-block:: python
+
+   from . import utilities as utils
+
+Apart from a function to easily save data, this module contains different kinds
+of small fixes to the data attributes, coordinates, and metadata which are 
+necessary for the data field to be CMOR-compliant.
+
+Note that this specific CMORizer script contains several subroutines in order
+to make the code clearer and more readable (we strongly recommend to follow
+that code style). For example, the function ``_get_filepath`` converts the raw
+filepath to the correct one and the function ``_extract_variable`` extracts and
+saves a single variable from the raw data.
+
+.. _utilities.py: https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/utilities.py
+
+
+4.2 Cmorizer script written in NCL
+----------------------------------
+
+Find here an example of a cmorizing script, written for the ``ESACCI XCH4``
+dataset that is available on the Copernicus Climate Data Store:
+`cmorize_obs_CDS-XCH4.ncl
```

Comment:
Note that the names have changed, they are now all lower case and - has been replaced by _

Reply:
Fix this so that the changes are mirrored here in the description.